reverse-proxy.md

```diff
@@ -255,6 +258,9 @@ server {
 
     listen 443 ssl http2;
     listen [::]:443 ssl http2; # comment to disable IPv6
+    
+    # Solves slow upload speeds caused by http2
+    http2_body_preread_size 1048576;
```

Comment:
```
    listen 443 ssl http2;
    listen [::]:443 ssl http2; # comment to disable IPv6
```
Here we enable http2 for nginx.
For Synology I have no Idea, I have one, but I've never used the reverse proxy function.
For NPM:
- this line would be needed to be added into the "Advanced Config" tab
- we could create an PR to the npm project, adding this line to the nginx.conf file (But the original npm is nearly unmaintained)
- or it would be included in my npm fork (if I get the docker-build-push action running again)
But I would wait making changes to the npm documentation until we know what helps here: https://github.com/nextcloud/all-in-one/discussions/979#discussioncomment-4696269

Reply:
> And sorry for pinging you again!

No problem, I had my notifications set to all activities for this repository anyway.

Reply:
> No problem, I had my notifications set to all activities for this repository anyway.

Thanks and good to know!